argoverse/evaluation/detection_utils.py

```diff
@@ -0,0 +1,174 @@
+# <Copyright 2020, Argo AI, LLC. Released under the MIT license.>
+"""Detection utilities for the Argoverse detection leaderboard.
+
+Accepts detections (in Argoverse ground truth format) and ground truth labels
+for computing evaluation metrics for 3d object detection. We have five different,
+metrics: mAP, ATE, ASE, AOE, and DCS. A true positive for mAP is defined as the
+highest confidence prediction within a specified euclidean distance threshold
+from a bird's-eye view. We prefer these metrics instead of IoU due to the
+increased interpretability of the error modes in a set of detections.
```

Comment:
this docstring looks good. i would indicate also that this thing uses multiprocessing and caches the results somehow (json, pickle, or just return type, etc) so it will use every core you have available btw

Reply:
Should we put it in the utils specifically? You can use the utils without multiprocessing

Reply:
bump

Reply:
a note has been place in `eval_detection.py`